addons/mail/security/ir.model.access.csv

```diff
@@ -17,7 +17,10 @@ access_mail_channel_all,mail.group.all,model_mail_channel,,1,0,0,0
 access_mail_channel_user,mail.group.user,model_mail_channel,base.group_user,1,1,1,1
 access_mail_channel_partner_public,mail.channel.partner.public,model_mail_channel_partner,base.group_public,1,0,0,0
 access_mail_channel_partner_portal,mail.channel.partner.portal,model_mail_channel_partner,base.group_portal,1,1,1,1
-access mail_channel_partner_user,mail.channel.partner.user,model_mail_channel_partner,base.group_user,1,1,1,1
+access_mail_channel_partner_user,mail.channel.partner.user,model_mail_channel_partner,base.group_user,1,1,1,1
+access_mail_moderation_email_portal,mail.mail.moderation.email.portal,model_mail_moderation_email,base.group_portal,1,0,0,0

```

Comment:
Here you give public and portal users the right to read all info from moderation. This is a security issue. I don't think they should be able to access this kind of internal data.